### ISSUE TITLE
attention mask and the pad token id were not set

### ISSUE THREAD

**Issue by jasonjabbourFieldAI** [yang-zj1026/NaVILA-Bench#25]
Hi, when I run the vlm_server.py script I get the following error when I try to retrieve an action. I believe this is making the model use more VRAM than it typically should. Has anyone encountered this issue before?

```
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:128001 for open-end generation.
Model generation took 0.53 seconds
```

**Comment by jasonjabbour** [comment]
Fixed this by setting `pad_token_id=self.tokenizer.eos_token_id` within the `vlm_server.py` script 👍

```
output_ids = self.model.generate(
    input_ids,
    images=[image_tensor],
    do_sample=False,
    temperature=0,
    top_p=None,
    num_beams=1,
    max_new_tokens=512,
    use_cache=True,
    stopping_criteria=[stopping_criteria],
    pad_token_id=self.tokenizer.eos_token_id, # Add this
)
```
